vsintegration/src/FSharp.ProjectSystem.Base/Microsoft.VisualStudio.Package.Project.resx

```diff
@@ -168,7 +168,7 @@
   <data name="CannotAddFileThatIsOpenInEditor" xml:space="preserve">
     <value>A file with name '{0}' already exists and is open in an editor. Please give a unique name to the item you are adding, or delete the existing item first.</value>
   </data>
-  <data name="CanNotSaveFileNotOpeneInEditor" xml:space="preserve">
+  <data name="CanNotSaveFileNotOpenInEditor" xml:space="preserve">
```

Comment:
Oh, sigh. I just dropped a pair of these corrections because I ran into the localizer. I'll put them back and fix them up.

Happy to toss in this fix-up shortly.